src/routes/legislative-process/+page.svelte

```diff
@@ -138,4 +177,8 @@
 	hr {
 		@apply border-0 border-solid border-gray-20 border-t w-full box-border;
 	}
+
+	ul {
+		@apply list-disc list-inside;
```

Comment:
According to Figma, if the list item has multiple lines, it needs to start at the same vertical line like:

![image](https://github.com/wevisdemo/parliament-watch/assets/8860448/cdd21101-468f-4cb4-a0b7-92794f0c0fe5)

So you need the `list-outside` with some left margin.